pkg/store/bucket_test.go

```diff
@@ -2318,13 +2328,13 @@ func BenchmarkBlockSeries(b *testing.B) {
 	head, _ := storetestutil.CreateHeadWithSeries(b, 0, storetestutil.HeadGenOptions{
 		TSDBDir:          filepath.Join(tmpDir, "head"),
 		SamplesPerSeries: 86400 / 15, // Simulate 1 day block with 15s scrape interval.
+		ScrapeInterval:   15 * 1000,
```

Comment:
Let's either use time.Duration or put unit in var name 🤗

Reply:
Converted to time.Duration